frontend/src/scenes/dashboard/dashboardsLogic.ts

```diff
@@ -25,6 +26,18 @@ export const dashboardsLogic = kea<dashboardsLogicType>({
                     .filter((d) => !d.deleted)
                     .sort((a, b) => (a.name ?? 'Untitled').localeCompare(b.name ?? 'Untitled')),
         ],
+        dashboardTags: [
+            () => [dashboardsModel.selectors.dashboards],
+            (dashboards: DashboardType[]): string[] =>
+                dashboards.length
+                    ? uniqueBy(
+                          dashboards
+                              .map((dashboard) => dashboard.tags.map((tag) => tag))
```

Comment:
This chain can be simplified to:

```
dashboard.flatMap(({tags} => tags)
```

I'd also consider adding a default argument to `uniqueBy`

Reply:
Magic, thanks for the tip!